test_files/enums.d

```diff
@@ -37,10 +37,11 @@ struct C
     _Anonymous_0 point;
 }
 
-enum _Anonymous_1
+alias i = C._Anonymous_0.i;
+alias j = C._Anonymous_0.j;
```

Comment:
It is problematic:
[C](https://ideone.com/YIkSfE) vs [C++](https://ideone.com/gB6HgD).